<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editmbikedialog</class>
 <widget class="QDialog" name="editmbikedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit motorbike</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_MB"/>
    </item>
    <item>
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Owner Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owname_MB"/>
    </item>
    <item>
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>Owner Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ownsur_MB"/>
    </item>
    <item>
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>Owner Civic ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ownid_MB"/>
    </item>
    <item>
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>Maximal Speed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="maxspe">
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_61">
      <property name="text">
       <string>Swept Volume</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="swepvo">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editmbikedialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editmbikedialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
